src/Views/scstormrunoff.html

```diff
@@ -135,66 +134,125 @@ <h4>Citation</h4>
+                                        <ng-form name="PRF" class="hidden-print">
+                                            <label for="landUse">Land Use<span style="color: #ffcc00">*</span></label>
+                                            <select name="landUse" ng-model="vm.prfForm.landUse" ng-options="item.name for item in vm.prfTypes" ng-change="vm.setPRF(vm.prfForm.landUse)" required></select>                                                                          
+
+                                            <label for="prfValue">PRF<span style="color: #ffcc00">*</span></label>
+                                            <input type="number" name="prfValue" ng-model="vm.prfForm.prfValue" required/>
+                                            <label for="area">Area (ac)<span style="color: #ffcc00">*</span></label>
+                                            <input type="number" name="area" ng-model="vm.prfForm.area" required/>
+                                            <div>&nbsp;</div>
+                                            <div class="form-row-extremes">
+                                                <div class="button-column-left">
+                                                    <button ng-click="vm.closeAddFlowSegment()"><i class="fa fa-times"></i> Cancel</button>
+                                                </div>
+                                                <div class="button-column-right">
+                                                    <button ng-disabled="PRF.$invalid" ng-click="vm.addFlowSegment()"><i class="fa fa-plus"></i> Add</button>
+                                                </div>
+                                            </div>
+                                            <div>&nbsp;</div>
+                                        </ng-form>
                                     </div>
                                 </div>
-                                
+                                <label for="PeakRateFactor">Peak Rate Factor, PRF:</label> <span ng-if="SyntheticUrbanHydrograph.PeakRateFactor.$error.pattern" class="red label"><span class="fa fa-exclamation-triangle"></span>&nbsp;Value must be greater than or equal to 0</span>
+                                <input type="number" name="PeakRateFactor" ng-model="vm.peakRateFactor" required readonly/>&nbsp;&nbsp;
+
+
+                                <!-- Time of Concentration -->
+                                <h3 class="center-div">Time of Concentration</h3>
+                                <div class="wim-alert hidden-print">Select a 'Time of concentration'. If 'Lag Time Equation' is selected, no additional user input is needed. If 'Travel Time Method' is selected user must add at least one flow segment for each flow type. Once filled out 'Time of Concentration' will be calculated automatically with <b>Calculate Missing Parameters</b>.</div>
```

Comment:
Few changes: 

Select a Time of Concentration`.  If 'Lag Time Equation' is selected, no additional user input is needed. If 'Travel Time Method' is selected, add at least one flow segment. Once filled out, 'Time of Concentration' will be calculated automatically with <b>Calculate Missing Parameters</b>.